util-images/request-benchmark/main.go

```diff
@@ -36,22 +36,65 @@ import (
 
 const (
 	HealthCheckRequests = 10
-	NamespaceTmpl       = "%namespace%"
+	NamespaceTmpl      = "%namespace%"
+)
+
+type ContentType string
```

Comment:
I wouldn't name the parameter `--content-type` unless it takes an actual media type as an value (eg `application/yaml`).